script.sh

```diff
@@ -85,9 +88,41 @@ else
   BUNDLE_EXEC="bundle exec "
 fi
 
+if [ "${INPUT_ONLY_CHANGED}" = "true" ]; then
+  echo '::group:: Getting changed files list'
+
```

Comment:
How about running `git fetch --depth 1 origin "${BASE_REF}" here?

Reply:
Seems like a good idea, compared to adding a step to do it manually. Only concern is if remote is named differently or if checkout is very custom, so the command breaks.

Reply:
Doing it after checking commit availability should be the good way to make it simple by default and solvable if default doesn't work: eb0eda02f00e11b2b1146c74d64457df726cb213